### ISSUE TITLE
VA Client outputs blank lines if watermarking enabled in spatial analytics pipelines	

### ISSUE THREAD

**Issue by whbruce** [intel/video-analytics-serving#80]
Spatial analytics extensions add regions for display in watermarked output which vaclient interprets as null detections.

Steps to re-create:
* Update vaclient/parameter_files/object-line-crossing.json to enable watermarking
* Re-build container
* Start service
* Run client as shown below and note output
```
$ ./vaclient/vaclient.sh run object_tracking/object_line_crossing https://github.com/intel-iot-devkit/sample-videos/blob/master/people-detection.mp4?raw=True --parameter-file vaclient/parameter_files/object-line-crossing.json
Pipeline running: object_tracking/object_line_crossing, instance = 1
Timestamp 0
-
-
-
-
-
-
-
-
-
Timestamp 83333333
<snip>
Timestamp 2000000000
- person (0.80) [0.31, 0.55, 0.48, 1.00] {'id': 1}
-
-
-
-
-
-
-
-
-
Timestamp 2083333333
- person (0.91) [0.32, 0.54, 0.47, 1.00] {'id': 1}
-
-
-
-
-
-
-
-
-
```

**Comment by antoniomtz** [comment]
I also experienced an issue when enabling watermark to the object zone count.
For some reason, I was not able to consume the RTSP stream on VLC. 
I was able to consume the RTSP stream on VLC if watermark is disable.

**Comment by whbruce** [comment]
@antoniomtz we are addressing and "extra line" issue. I will open a separate issue for zone counting watermarking preventing RTSP re-streaming with VLC.

**Comment by whbruce** [comment]
Fixed in [v0.6.1](https://github.com/intel/video-analytics-serving/releases/tag/v0.6.1-beta).
